x-pack/plugins/ml/public/ui_actions/edit_swimlane_panel_action.tsx

```diff
@@ -39,20 +45,23 @@ export function createEditSwimlanePanelAction(
           '../embeddables/anomaly_swimlane/anomaly_swimlane_setup_flyout'
         );
 
-        const result = await resolveAnomalySwimlaneUserInput(
-          coreStart,
-          deps.data.dataViews,
-          embeddable.getInput()
-        );
-        embeddable.updateInput(result);
+        const { jobIds, viewBy, swimlaneType, panelTitle } = context.embeddable;
+
+        const result = await resolveAnomalySwimlaneUserInput(coreStart, deps.data.dataViews, {
+          jobIds,
+          swimlaneType: swimlaneType.getValue(),
+          viewBy: viewBy.getValue(),
+          title: panelTitle?.getValue(),
```

Comment:
Any edit I make to the panel title via the Edit swim lane action isn't getting applied.

Reply:
Fixed in https://github.com/elastic/kibana/pull/179073/commits/f5b541480fe8bf47f575a6b481d1402f16142d5d